### PR TITLE
Bug 1113396 - Make unclassified jobs visible in only-unclassified mode

### PR DESCRIPTION
**Pull request by dewgeek**
after deleting classification , it's failure classification id is changed to 1 , which makes it unclassified error

### CODE REVIEW

**Comment by tojonmz** [comment]
Same here with the commit message, as with https://github.com/mozilla/treeherder-ui/pull/391#issuecomment-76292586. Probably something like: `Bug 1113396 - Make unclassified jobs visible in only-unclassified mode`

File: webapp/app/plugins/annotations/controller.js [hunk 1]
```diff
@@ -21,6 +21,14 @@ treeherder.controller('AnnotationsPluginCtrl', [
         }, true);
 
         $scope.deleteClassification = function(classification) {
+
+            var key = "key" + classification.job_id;
+            var jobMap = ThResultSetStore.getJobMap($rootScope.repoName);
+            var job = jobMap[key].job_obj;
+
+            job.failure_classification_id = 1 ;
```
Comment (tojonmz) [inline webapp/app/plugins/annotations/controller.js]: There no space needed after the `1` and the `;`

File: webapp/app/plugins/annotations/controller.js [hunk 2]
```diff
@@ -21,6 +21,14 @@ treeherder.controller('AnnotationsPluginCtrl', [
...
+            ThResultSetStore.updateUnclassifiedFailureMap($rootScope.repoName, job)
```
Comment (tojonmz) [inline webapp/app/plugins/annotations/controller.js]: I think this needs a trailing `;`

**Comment by dewgeek** [comment]
@tojonmz , I squashed my changes to avoid multiple commits during merge

**Comment by tojonmz** [comment]
@dewgeek cool, just a note that I think you'll typically only squash right at the end, once the main reviewer (eg. @camd) has approved the entire PR and all the changes. Although Github does show the outdated diffs in the conversation history, they lose the series of commits in the commits tab because of the squash. I believe normally you only want to blow those away with a squash once, right at the end, before the merge.

It could vary with with the repo owners though. It's just my general understanding of best practice.

**Comment by tojonmz** [comment]
Oh, and the commit message still needs to be updated, per my first comment in the conversation above.

**Comment by dewgeek** [comment]
@tojonmz , commit message ammended :)

**Comment by camd** [comment]
looks good!  Works well.  thanks  @dewgeek!!  :)

**Pull request merged**